doc/api/path.md

```diff
@@ -523,6 +526,11 @@ On Windows:
 // Returns: ['foo', 'bar', 'baz']
 ```
 
+*Note*: On Windows, both the forward slash (`/`) and backward slash (`\`) are
+accepted as path segment separators; however, if separators are to be added by
+Windows-specific implementations of the `path` methods, only the backward slash
+(`\`) will be used.
```

Comment:
I'd change the part after the semicolon to "the `path` methods only add backward slashes (`\`)."

"if ... to be ..." is probably hard to follow for people without a good command of English.